### ISSUE TITLE
Error Rebuilding index after new document loaded from zip file...

### ISSUE THREAD

**Issue by bvahedy** [AndBible/and-bible#2682]
Affects latest beta: 

After loading an existing bible from the zip file. 
Tried to search and the app asked to rebuild index.  After accepting to rebuild the progress bar goes almost to the end but errors after 35 seconds. 
On the second attempt after closing the app and reopening it, it seems to work.

**Comment by tuomas2** [comment]
is this consistent? Have you tried many times with multiple different documents?

**Comment by bvahedy** [comment]
Yes, it's consistent.  In main bible view I have NASB, then I go to download document and find NASB, then I tap to reinstall it. After the install I go to the find screen and tap on the kebab menu and tap on rebuild. Error happens. 

I tried NASB and HCB, they both do the same thing. 
![Screenshot_20230727_083106_Bible Study](https://github.com/AndBible/and-bible/assets/46405809/1fc0a241-c946-45d9-9d27-0e90e094430d)
